pkg/cli/commands.go

```diff
@@ -21,6 +21,7 @@ func AddCommands(cmd *cobra.Command, cli CLI) {
 
 	clusterCmd := NewClusterCmd(cli)
 	clusterCmd.AddCommand(NewClusterNodesMissingImageCmd(cli))
+	clusterCmd.AddCommand(NewClusterFreeDiskSpaceCmd(cli))
```

Comment:
The name/UX here does not seems intuitive for me:

> kurl cluster free-disk-space [--storageclass openebs] [--openebs-node node] [--openebs-image eeacms/rsync:2.3]

Will it ensure that we have a free disk space or is it checking validating that we have a free space?

Therefore, I understand that we have many checks in commands such this one.
So, we could looking for in the future how we could better organize and etc the subcommands.

However, for now could we call this one check-free-disk-space? Is that OK? WDYT?

Reply:
fixed.